### ISSUE TITLE
Table being Scrambled when PDF saves

### ISSUE THREAD

**Issue by Tsurumah** [stolksdorf/homebrewery#221]
**Browser Type/Version**: Version 54.0.2840.71 m (64-bit)

**Issue Description**: Everything displays correctly--including when I go to the "Print" page; however, when the PDF is done printing, every table appears to be scrambled completely. It only started this the last week or so, and does so on both Chrome and Canary (Version 56.0.2905.0 canary (64-bit)), and on every document that I've made thus far, including ones that are months old and were finished.

EDIT: I also noticed that it does the same thing to sidebars as well. [Imgur](http://i.imgur.com/lCEcv8f.jpg)

**Brew code to reproduce**: <details><summary>

Click to expand code</summary>

```
<div class='classTable wide'>
##### Anvelstad Character Table
| &nbsp;&nbsp;&nbsp;Level&nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Class&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;HP&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Proficiency Bonus | Features |
|:---:|:--------:|:---:|:---:|:---:|
| 1st | Rogue 1 | 8 | +2 | Expertise (INVESTIGATION, PERCEPTION); Sneak Attack (1d6), Observant |
| 2nd | Rogue 2 | 14 |  +2 | Cunning Action |
| 3rd | Rogue 3 | 19 |  +2 | Inquisitive Archetype: Ear for Deceit, Eye for Detail, Insightful Fighting; Sneak Attack (2d6)  |
| 4th | Rogue 4 | 24 |  +2 | Ability Score Improvement (+2 DEX) |
| 5th | Scholar 1 | 30 |  +3 | Multiclass; Ingenuity; Field of Study: Physician | 
| 6th | Scholar 2 | 37 |  +3 | Expertise (INSIGHT, STEALTH); Erudite Applications (DOGGED, BONUS PROFICIENCIES [ARCANA, PERSUASION]) |
| 7th | Scholar 3 | 42 |  +3 | Well Traveled |
| 8th | Scholar 4 | 48 | +3 | Ability Score Improvement (+2 INT) |
| 9th | Scholar 5 | 56 | +4 | Font of Ingenuity; Erudite Application (EXTRA ATTACK) |
| 10th | Scholar 6 | 64 |  +4 | Fiend of Study Feature |
| 11th | Scholar 7 | 69 |  +4 | Erudite Application (ELDRITCH LORE) |
| 12th | Scholar 8 | 76 |  +4 | Ability Score Improvement (+2 INT) |
| 13th | Scholar 9 | 84 |  +5 | Field Research; Erudite Application (KEEN SENSES) |
| 14th | Scholar 10 | 89 |  +5 | Expertise (ARCANA, PERSUASION) |
| 15th | Scholar 11 | 97 |  +5 | Erudite Application (INTELLECTUAL PRECISION) | 
| 16th | Scholar 12 | 104 |  +5 | Ability Score Improvement (+2 DEX) |
| 17th | Scholar 13 | 109 |  +6 | Field of Study Feature |
| 18th | Scholar 14 | 114 |  +6 | Erudite Application (PSYCHOLOGICAL PROFILE) |
| 19th | Scholar 15 | 119 |  +6 | Analytical Mind |
| 20th | Scholar 16 | 124 |  +6 | Ability Score Improvement (+1 INT, +1 CON) |
</div>
```

</details>

**Related Images** : [Imgur](http://i.imgur.com/hh6YQuU.jpg)

**Comment by Tsurumah** [comment]
Update: It appears to only be this computer, which is a bummer because it's my main. My wife's computer displays and prints it fine.

**Comment by calculuschild** [comment]
This has been solved! See #683.
